keel-plugin/src/main/kotlin/com/netflix/spinnaker/keel/plugin/DefaultResourceValidator.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.keel.plugin
+
+import com.netflix.spinnaker.keel.api.Resource
+import org.springframework.stereotype.Component
+import kotlin.reflect.KClass
+
+@Component
+class DefaultResourceValidator : ResourceValidator<Any> {
```

Comment:
I'm not sure this provides anything useful other than ensuring there is at least one ResourceValidator

The default implementation of validate in ResourceHandler does an objectMapper conversion of the spec to the generic type which would be useful if we could get it in here (but the typing bit of that might be tricky to pull off)

Reply:
Removing it, you are right. Was just using it for testing.